client/src/app/dashboard/dashboard.component.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { DashboardComponent } from './dashboard.component';
+
+describe('DashboardComponent', () => {
+  let component: DashboardComponent;
+  let fixture: ComponentFixture<DashboardComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ DashboardComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(DashboardComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  // Default Option
+  it('#onSelect should set #selectedOption to "The First option"', () => {
+    expect(component.selectedOption.title).toBe('Create My Account', 'Initial Option');
+  });
+
+  // Tests Options are clickable
+  it('should test click 2nd option', () => {
```

Comment:
I think this test name is wrong. I see you are clicking on all elements and not just the second one.

Reply:
You are correct, the code was changed to test them all, comment has been removed in latest commit as per your request:
c2cb3c8c06ac7c0e18bbbba34240c8047d1f068e